test/BaseClass.cpp

```diff
@@ -0,0 +1,53 @@
+#define Adafruit_MAX31865_Test Adafruit_MAX31865_Base
+#include "Common.cpp"
+
+unittest(testingClassName) {
+  std::cout << std::endl;
```

Comment:
Once you finish debugging things, do you still need the references to `std::cout`? If they are useful for when `showData` is `true` in `BitCollector` then maybe you could have a `#define` that turns the debugging on or off.